### ISSUE TITLE
Add plugin to WordPress.org plugin repository

### ISSUE THREAD

**Issue by hajekj** [psignoret/aad-sso-wordpress#186]
I think it would be really cool to have this plugin available in the official WordPress plugin repository. Any way I could help with that?

Previous issues: #60 #150

**Comment by psignoret** [comment]
I agree! I would enthusiastically welcome a PR with an initial [readme.txt](https://wordpress.org/plugins/readme.txt).

**Comment by hajekj** [comment]
So I suppose next thing that follows should be following - https://wordpress.org/plugins/developers/add/

**Comment by hajekj** [comment]
Hey @psignoret, is there any progress on this one?
